streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -459,7 +459,6 @@ void handleLostAll() {
         final Iterator<Task> iterator = tasks.values().iterator();
         while (iterator.hasNext()) {
             final Task task = iterator.next();
-            final Set<TopicPartition> inputPartitions = task.inputPartitions();
```

Comment:
Is this change intentional?

Reply:
Yep, sorry I meant to leave a comment. I noticed this is redundant since `cleanupTask` (invoked by `closeTaskDirty`) does the same thing